packages/truffle-db/src/loaders/index.ts

```diff
@@ -26,6 +26,7 @@ const resolvers = {
     artifactsLoad: {
       resolve: async (_, args, { artifactsDirectory, contractsDirectory, workingDirectory, db }, info) => {
         const tempDir = tmp.dirSync({ unsafeCleanup: true })
+        tmp.setGracefulCleanup();
```

Comment:
Does this also need that `removeCallback()` business?

Reply:
Yep, it's there on line 39 from before!